functions/stare/Makefile.am

```diff
@@ -0,0 +1,43 @@
+
+
+AUTOMAKE_OPTIONS = foreign 
+
+# Headers in 'tests' are used by the arrayT unit tests.
+
+#AM_CPPFLAGS = $(BES_CPPFLAGS) -I$(top_srcdir)/dispatch -I$(top_srcdir)/dap -I$(top_srcdir)/functions $(DAP_CFLAGS) $(GDAL_CFLAGS)
+#AM_LDADD =  $(DAP_SERVER_LIBS) $(GF_LIBS) $(top_builddir)/dispatch/libbes_dispatch.la
+
+
+#Define __macosx for STARE
+#Remove hard code for filepath
+AM_CPPFLAGS = $(DAP_CFLAGS) -D__macosx -I$(top_srcdir)/../hyrax-dependencies/src/STARE/include -I$(prefix)/deps/include
```

Comment:
This line and the one below need to be changed. The two values of -I are both not going to work in general. For most builds $prefix is not set and hyrax-dependencies won't be anywhere predictable.